### DISCUSSION TITLE
DayZ Server installs DayZ Experimental server, and not the ordinary server

### DISCUSSION THREAD

**Discussion by christophersteinsvik** [GameServerManagers/LinuxGSM#3770] [Q&A] [answered]
Hello,

I have noticed that DayZ installs the DayZ Experimental Test Server ( https://steamdb.info/app/1042420/ ), and I believe it is supposed to install the DayZ Server ( https://steamdb.info/app/223350/ )

As a result of this, I am unable to connect to the server from the DayZ client due to the server being a different build than the client. I haven't tried, but I assume I will be able to connect to the server, if I launch the DayZ Experimental launcher...

Is this by design, or by accident?

**Comment by Claiyc** [comment, marked as answer]
DayZ Server is windows only. DayZ Experimental Test Server supports Linux.

**Reply by airtonix** [reply]
This isn't true anymore
